### DISCUSSION TITLE
支持西部数码的域名做ddns吗

### DISCUSSION THREAD

**Discussion by libix-cola** [jeessy2/ddns-go#959] [Q&A] [answered]
大佬  我只有v6的公网，能不能支持西部数码的域名啊

**Comment by libix-cola** [comment, marked as answer]
已经解决了，如果有需要的可以问我

**Comment by woshichenghaibo** [comment]
> 大佬 我只有v6的公网，能不能支持西部数码的域名啊

请教如何实现 的

**Comment by FulLiunian** [comment]
西部数据有文档支持

woshichenghaibo ***@***.***> 于2024年3月28日周四 10:07写道：
